src/cogent3/core/alignment.py

```diff
@@ -2774,15 +2774,15 @@ def _get_raw_pretty(self, name_order):
 
     def _repr_html_(self):
         html = self.to_html(
-            name_order=self.names[: self._repr_policy["num_seqs"]],
```

Comment:
should not be deleting `name_order`. This specifies the row order in the display

Reply:
Hi Gavin, `name_order` isn't deleted, I came up a fix that lets method `to_html`(a little bit below this code snippet) take `num_seqs` as an input parameter instead of `name_order` (sliced self.names), then method `to_html` will produce `name_order` inside it's logic. specifically, after 
`name_order = list(self.names)`
`name_order.remove(ref)`
`name_order.insert(0, ref)` 
Please let me know any issues!

Reply:
Apologies that I forgot updating the function docstring accordingly, let me know what do you think Gavin if you are happy with this fix, I am gonna push my updated docstring. cheers!

Reply:
But `name_order` is part of the `to_html()` public API and an argument users will want to be able to set. So better for the `_repr_html_ ()` calling code provide the correct number of names.